shared/login/signup/error/container.tsx

```diff
@@ -1,21 +1,37 @@
 import Error from '.'
 import {connect} from '../../../util/container'
+import {networkErrorCodes} from '../../../util/errors'
+
 import * as SignupGen from '../../../actions/signup-gen'
 
 type OwnProps = {}
 
 const mapStateToProps = state => ({
-  error: state.signup.signupError.stringValue(),
+  error: state.signup.signupError,
 })
 
 const mapDispatchToProps = dispatch => ({
   onBack: () => dispatch(SignupGen.createGoBackAndClearErrors()),
 })
 
+const mergeProps = (stateProps, dispatchProps) => {
+  let header = 'Ah Shoot! Something went wrong, try again?'
+  let body = stateProps.error ? stateProps.error.desc : ''
+  if (!!stateProps.error && networkErrorCodes.includes(stateProps.error.code)) {
+    header = "Our servers didn't respond. Try again?"
```

Comment:
@cecileboucheron should we mention servers in the error messages? it might confuse some users..